src/components/contactBottomHeaderArea.scss

```diff
@@ -0,0 +1,173 @@
+.contact .header-area .top {
+    margin: var(--spacing-s) 0;
+    border-bottom: none;
+}
+
+.contact .header-area .bottom {
+    margin-top: 48px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.contact .header-area .bottom .bottom__left {
+    flex: 0 1 60%;
+}
+
+.contact .header-area .bottom .bottom__left .form-row {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 52px;
+}
+
+.contact .header-area .bottom .bottom__left .form-row:last-of-type {
+    margin-bottom: 18px;
+}
+
+.contact .header-area .bottom .bottom__left .form-row>* {
+    width: 48%;
+}
+
+.contact .header-area .bottom .bottom__left .form-row input {
```

Comment:
Can we add some left padding to this element when it is focused? Otherwise the start of the text overlays with the focus border.

![image](https://user-images.githubusercontent.com/22242017/102108606-bef76480-3df8-11eb-94a2-ba138a16898f.png)